irohc/readme.md

```diff
@@ -0,0 +1,8 @@
+# C bindings for Iroh
+
+Running `cargo build --release` will produce a dynamic libaray. `irohc.h` is a handcrafted header file for that library. Should work on all platforms, I've only tested it on MacOS so far.
+
+For builds targeting older versions of MacOS, build with with:  `MACOSX_DEPLOYMENT_TARGET=10.7 && cargo build --target x86_64-apple-darwin --release`.
+
```

Comment:
I would recommend to use https://docs.rs/safer-ffi/latest/safer_ffi/ it refuces safety and maintenance issues considerably